File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT2_JAVA/src/Organisms/Plants/Berry.java
package Organisms.Plants;

import Game.World;

import java.awt.*;
import java.io.Serializable;

public class Berry extends Plant implements Serializable {
    public Berry(World world, int x, int y)
    {
        super("B", "Berry", new Color(142, 13, 255), 99, 0, 1, world,x,y);
    }
    @Override
    public Berry getChild(int x, int y)
    {
        return new Berry(world,x,y);
    }
}

File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT3_PYTHON/Organisms/Plants/berry.py
from Organisms.Plants.plant import Plant


class Berry(Plant):
    def __init__(self, position_x, position_y, world):
        super().__init__(position_x, position_y, "Jagoda", 'B', 99, 0, world, 0, (139,0,139))

    def getchild(self, newposition):
        return Berry(newposition[0], newposition[1], self.world)
